src/appkit.rs

```diff
@@ -2211,3 +2211,310 @@ impl NSScreen for id {
         msg_send_stret()(self, selector("convertRectToBacking:"), aRect)
     }
 }
+
+#[repr(u64)]
+pub enum NSTabViewType {
+    NSTopTabsBezelBorder     = 0,
+    NSLeftTabsBezelBorder    = 1,
+    NSBottomTabsBezelBorder  = 2,
+    NSRightTabsBezelBorder   = 3,
+    NSNoTabsBezelBorder      = 4,
+    NSNoTabsLineBorder       = 5,
+    NSNoTabsNoBorder         = 6
+}
+
+pub trait NSTabView {
+    unsafe fn new(_: Self) -> id {
+        msg_send()(class("NSTabView"), selector("new"))
+    }
+    unsafe fn initWithFrame_(self, frameRect: NSRect) -> id;
+    unsafe fn addTabViewItem_(self, tabViewItem: id);
+    unsafe fn insertTabViewItem_atIndex_(self,tabViewItem:id, index:NSInteger);
+    unsafe fn removeTabViewItem_(self,tabViewItem:id);
+
+    unsafe fn indexOfTabViewItem_(self, tabViewItem:id) -> id;
+    unsafe fn indexOfTabViewItemWithIdentifier_(self,identifier:id) -> id;
+    unsafe fn numberOfTabViewItems(self) -> id;
+    unsafe fn tabViewItemAtIndex_(self,index:id) -> id;
+
+    unsafe fn tabViewItems(self) -> id;
+
+    unsafe fn selectFirstTabViewItem_(self,sender:id);
+    unsafe fn selectLastTabViewItem_(self,sender:id);
+    unsafe fn selectNextTabViewItem_(self, sender:id);
+    unsafe fn selectPreviousTabViewItem_(self,sender:id);
+    unsafe fn selectTabViewItem_(self,tabViewItem:id);
+    unsafe fn selectTabViewItemAtIndex_(self,index:id);
+    unsafe fn selectTabViewItemWithIdentifier_(self,identifier:id);
+    unsafe fn selectedTabViewItem(self) -> id;
+    unsafe fn takeSelectedTabViewItemFromSender_(self,sender:id);
+
+    unsafe fn font(self) -> id;
+    unsafe fn setFont_(self, font:id);
+
+    unsafe fn tabViewType(self) -> NSTabViewType;
+    unsafe fn setTabViewType_(self,tabViewType: NSTabViewType);
+
+    unsafe fn controlTint(self) -> id;
+    unsafe fn setControlTint_(self,controlTint:id);
+
+    unsafe fn drawsBackground(self) -> BOOL;
+    unsafe fn setDrawsBackground_(self,drawsBackground:BOOL);
+
+    unsafe fn minimumSize(self) -> id;
+    unsafe fn contentRect(self) -> id;
+    unsafe fn controlSize(self) -> id;
+    unsafe fn setControlSize_(self,controlSize:id);
+    
+    unsafe fn allowsTruncatedLabels(self) -> BOOL;
+    unsafe fn setAllowsTruncatedLabels_(self, allowTruncatedLabels:BOOL);
+
+    unsafe fn setDelegate_(self, delegate:id);
+    unsafe fn delegate(self) -> id ;
+
+    unsafe fn tabViewAtPoint_(self, point:id) -> id;
+
+
+}
+
+impl NSTabView for id {
+
+    unsafe fn initWithFrame_(self, frameRect: NSRect) -> id {
```

Comment:
Would you be able to document the `id` types like I have here? https://github.com/bjz/rust-cocoa/commit/c4b1941432a43acbf9492aea74629f88a8b2c72c#diff-8550f09508786ee2e01c09c7d4722142R1311